### ISSUE TITLE
[Issue]: ChunkLoadError: Loading chunk failed. Blank UI with errors in browser console

### ISSUE THREAD

**Issue by ramkrishna757575** [jellyfin/jellyfin#10769]
### Please describe your bug

When opening the Jellyfin web UI in the browser, the background color of the page and navbar loads, but all content and everything is just blank. The page URL is `http://192.168.1.175:8096/jellyfin/web/index.html#!/login.html?serverid=3db78fcbab88484f94c7944a485534d5`
**Note**: Base URL is set as `/jellyfin`

This is happening in browser where jellyfin was not already logged-in. In the browser, where user is already logged-in, the issue doesn't exit and UI loads properly.
This also doesn't occur in the Jellyfin Media Player Windows App (in a fresh install, I am able to login and add the server)

### Jellyfin Version

Other

### if other:

10.8.13

### Environment

```markdown
- OS: Ubuntu 22.04
- Linux Kernel: 6.2.0-39-generic
- Virtualization: Docker
- Clients: Browser, Windows 11, MAC OS
- Browser: Brave
- FFmpeg Version: 5.1.4-Jellyfin
- Playback Method: Direct Play, Transcode
- Hardware Acceleration: NVENC
- GPU Model: GTX1080
- Plugins: 
- Reverse Proxy:
- Base URL: /jellyfin
- Networking: Bridge
- Storage: local
```

### Please attach any browser or client logs here

GET http://192.168.1.175:8096/jellyfin/web/session-login.c88fc4019dc5f2693ced.chunk.js net::ERR_ABORTED 404 (Not Found)

Uncaught (in promise) ChunkLoadError: Loading chunk 7051 failed.
(error: http://192.168.1.175:8096/jellyfin/web/session-login.c88fc4019dc5f2693ced.chunk.js)
    at i.f.j (main.jellyfin.bundle.js?dd8dd0da90b7b671c030:2:1206058)
    at main.jellyfin.bundle.js?dd8dd0da90b7b671c030:2:1189876
    at Array.reduce (<anonymous>)
    at i.e (main.jellyfin.bundle.js?dd8dd0da90b7b671c030:2:1189841)
    at Array.map (<anonymous>)
    at o (main.jellyfin.bundle.js?dd8dd0da90b7b671c030:2:1179495)
    at e.value (main.jellyfin.bundle.js?dd8dd0da90b7b671c030:2:19826)
    at main.jellyfin.bundle.js?dd8dd0da90b7b671c030:2:19664
    at e.value (main.jellyfin.bundle.js?dd8dd0da90b7b671c030:2:23653)
    at e.value (main.jellyfin.bundle.js?dd8dd0da90b7b671c030:2:19633)

Error opening web socket: Error: Cannot open web socket without access token.

### Please attach any screenshots here

![Screenshot 2023-12-25 203658](https://github.com/jellyfin/jellyfin/assets/14249848/649b4912-b143-461d-ad0e-c5cd4a3cd531)
![Screenshot 2023-12-25 203720](https://github.com/jellyfin/jellyfin/assets/14249848/19c417b4-e014-4fa5-be47-d7176e5b2e24)
![Screenshot 2023-12-25 203956](https://github.com/jellyfin/jellyfin/assets/14249848/daf400db-1d76-4d7f-b9f1-8aa240c73e11)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by felix920506** [comment]
Hi, please provide server logs. Also, the IP you provided is a LAN IP and not accessible from outside your local network.

**Comment by ramkrishna757575** [comment]
Sorry I did not get time to reply to this. Got busy in travelling.

I found the cause of the issue. It was happening because I was using the intro-skipper plugin. And to get the skip button in the web UI, I had followed the guide to replace the original Jellyfin webui source with the one provided by the plugin.

Turns out the plugin is no longer maintained. So it was causing the issue and not serving the correct frontend files.
Once I removed it, everything went back to normal.
